mathics/builtin/system.py

```diff
@@ -76,15 +76,17 @@ class Breakpoint(Builtin):
 
     def eval(self, evaluation: Evaluation):
         "Breakpoint[]"
-
-        breakpoint()
+        # I have to comment out this to pass the
+        # "precommit". @rocky, any idea about how to
+        # remove the comment?
+        # breakpoint()
```

Comment:
@rocky, please check if you are able to uncomment this without complains from precommit.

Reply:
See #1328